documentation/dev-portal/src/clients/socks5/setup.md

```diff
@@ -100,21 +36,19 @@ Before you can use the client, you need to initalise a new instance of it, which
-
-The `--provider` field needs to be filled with the Nym address of a Network Requester that can make network requests on your behalf. If you don't want to [run your own](../nodes/network-requester.md) you can select one from the [mixnet explorer](https://explorer.nymtech.net/network-components/service-providers) by copying its `Client ID` and using this as the value of the `--provider` flag. Alternatively, you could use [this list](https://harbourmaster.nymtech.net/).
+The `--use-reply-surbs` field denotes whether you wish to send [SURBs](https://nymtech.net/docs/architecture/traffic-flow.md#private-replies-using-surbs) along with your request. It defaults to `false`, we are explicitly setting it as `true`. It defaults to `false` for compatibility with older versions of the [Network Requester](https://nymtech.net/operators/nodes/network-requester.md).
 
-Since the nodes on this list are the infrastructure for [Nymconnect](https://nymtech.net/developers/quickstart/nymconnect-gui.html) they will support all apps on the [default whitelist](../nodes/network-requester.md#network-requester-whitelist): Keybase, Telegram, Electrum, Blockstream Green, and Helios.
+The `--provider` field needs to be filled with the Nym address of a Network Requester that can make network requests on your behalf. If you don't want to [run your own](https://nymtech.net/operators/nodes/network-requester.md) you can select one from the [mixnet explorer](https://explorer.nymtech.net/network-components/service-providers) by copying its `Client ID` and using this as the value of the `--provider` flag. Alternatively, you could use [this list](https://harbourmaster.nymtech.net/).
 
 #### Choosing a Gateway
 By default - as in the example above - your client will choose a random gateway to connect to.
 
 However, there are several options for choosing a gateway, if you do not want one that is randomly assigned to your client:
 * If you wish to connect to a specific gateway, you can specify this with the `--gateway` flag when running `init`.
 * You can also choose a gateway based on its location relative to your client. This can be done by appending the `--latency-based-selection` flag to your `init` command. This command means that to select a gateway, your client will:
-	* fetch a list of all availiable gateways
-	* send few ping messages to all of them, and measure response times.
-	* create a weighted distribution to randomly choose one, favouring ones with lower latency.
+    * fetch a list of all available gateways
+    * send few ping messages to all of them, and measure response times.
+    * create a weighted distribution to randomly choose one, favouring ones with lower latency.
```

Comment:
Maybe a little note on formatting - also a question for me. The bullet points above start with a capitol letter, these don't. some end with `.` some don't. 

Maybe there is some other logic more than a doctrine type convention, let me know and I will adapt.